content/02.delete-me.md

```diff
@@ -1,15 +1,146 @@
-## Manubot Rootstock Information
+## Introduction
 
-**Note: Manubot instances should delete this file.**
+This manuscript is a template/boilerplate (aka "rootstock") for [Manubot](https://manubot.org/ "Manubot"), a tool for writing scholarly manuscripts.
+Use this template as a starting point for your manuscript by following the directions on the [repository homepage](https://github.com/greenelab/manubot-rootstock "Manubot Rootstock").
```

Comment:
We should update this line to https://github.com/manubot/rootstock

Do we still have an example of a manual reference in this updated demo?  The previous example was to `url:https://github.com/greenelab/manubot-rootstock`, which we should also update.

Reply:
We have:

`URL citation [@url:https://greenelab.github.io/meta-review]`

Do you want to change it to the rootstock?